ArduCopter/ArduCopter.cpp

```diff
@@ -565,6 +574,7 @@ void Copter::update_GPS(void)
 #endif
         }
     }
+    barometer.update_gps_alt(float(gps.location().alt)/100.f);
```

Comment:
dividing is harder than multiplying and should be avoided where possible. Please change all the "/100.f" to "*0.01f"